apps/wallet/src/ui/app/pages/swap/MaxSlippage.tsx

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import BottomMenuLayout, { Content, Menu } from '_app/shared/bottom-menu-layout';
+import { Button } from '_app/shared/ButtonUI';
+import { Text } from '_app/shared/text';
+import { IconTooltip } from '_app/shared/tooltip';
+import { IconButton } from '_components/IconButton';
+import NumberInput from '_components/number-input';
+import Overlay from '_components/overlay';
+import { DescriptionItem } from '_pages/approval-request/transaction-request/DescriptionList';
+import { type FormValues } from '_pages/swap/utils';
+import { Settings16 } from '@mysten/icons/src';
+import { useField, useFormikContext } from 'formik';
+import { useState } from 'react';
+
+export function MaxSlippageContent({ setModalOpen }: { setModalOpen: (isOpen: boolean) => void }) {
+	const { values } = useFormikContext<FormValues>();
```

Comment:
I think we are trying to move away from formik but probably makes sense to use it for now

Reply:
We shouldn't be using formik on new components.

Reply:
Changed to react-hook-form